develop/parachain-devs/parachain-maintenance/runtime-upgrades.md

```diff
@@ -0,0 +1,81 @@
+| `spec_version`        | The version of the runtime specification. A full node won't attempt to use its native runtime in substitute for the on-chain WebAssembly runtime unless all of `spec_name`, `spec_version`, and `authoring_version` are the same between the WebAssembly and native binaries. Updates to the `spec_version` can be automated as a CI process, as is done for the [Polkadot network](https://gitlab.parity.io/parity/mirrors/polkadot/-/blob/master/scripts/ci/gitlab/check_extrinsics_ordering.sh){target=_blank}. This parameter is typically incremented when there's an update to the `transaction_version`.                      |
+| `impl_version`        | The version of the implementation of the specification. Nodes can ignore this. It is only used to indicate that the code is different. As long as the `authoring_version` and the `spec_version` are the same, the code itself might have changed, but the native and WebAssembly binaries do the same thing. In general, only non-logic-breaking optimizations would result in a change of the `impl_version`.                                                                                                                                                                                                    |
+| `transaction_version` | The version of the interface for handling transactions. This parameter can be useful to synchronize firmware updates for hardware wallets or other signing devices to verify that runtime transactions are valid. The parameter allows hardware wallets to know which transactions they can safely sign. This number must be bumped if there is a change in the index of the pallets in the `construct_runtime!` macro or if there are any changes to dispatchable functions, such as the number of parameters or parameter types. If this number is updated, then the `spec_version` must also be updated. |
+| `apis`                | A list of supported [runtime APIs](https://paritytech.github.io/polkadot-sdk/master/sp_api/macro.impl_runtime_apis.html){target=_blank} along with their versions.                                                                                                                                                                                                                                                                                                                                                                                                                                                            |
+The executor that the native runtime has the same consensus-driven logic as the WebAssembly before it chooses to execute it. However, because the runtime versioning is set manually, the executor can still make inappropriate decisions if the runtime version is misrepresented.
```

Comment:
```suggestion
| `apis`                | A list of supported [runtime APIs](https://paritytech.github.io/polkadot-sdk/master/sp_api/macro.impl_runtime_apis.html){target=_blank} along with their versions.                                                                                                                                                                                                                                                                                                                                                                                                                                                            |

The executor that the native runtime has the same consensus-driven logic as the WebAssembly before it chooses to execute it. However, because the runtime versioning is set manually, the executor can still make inappropriate decisions if the runtime version is misrepresented.
```